lib/std/io/in_stream.zig

```diff
@@ -231,6 +234,18 @@ pub fn InStream(
             }
         }
 
+        /// Reads `slice.len` bytes from the stream and returns if they are the same as the passed slice
+        pub fn isBytes(self: Self, slice: []const u8) !bool {
```

Comment:
Rename to readWithCheck or skipWithCheck?

Reply:
Or maybe `readEqual` or `readEql`?

Reply:
The first impression:
Q: stream.isBytes?
A: Sure, any stream is bytes.